Getting Started/WalkthroughFirstWPFApp/csharp/ExpenseItIntro.csproj

```diff
@@ -1,53 +1,17 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You should probably also add this to the larger solution file [here](https://github.com/microsoft/WPF-Samples/blob/master/WPFSamples.sln).

Reply:
good point